README.md

```diff
@@ -7,14 +7,10 @@
 # To build this application:
 
 ```
-$ mkdir wgo
-$ cd wgo
-$ export GOPATH=`pwd`
-$ export GOBIN=$GOPATH/bin
-$ export PATH=$GOBIN:$PATH
 $ go get github.com/skelterjohn/wgo
 $ go install github.com/skelterjohn/wgo
```

Comment:
@skelterjohn Please keep in mind that these repos are meant for experimental use by Glenn and I. That is why the instructions are so verbose. When we figure out where these examples will eventually live, they'll be cleaned up substantially.

The reason for all of the directory creation and environment variable setting was to demonstrate that this was a completely isolated development workspace/environment.